gensrc/thrift/StatusCode.thrift

```diff
@@ -85,6 +85,8 @@ enum TStatusCode {
     UNKNOWN = 50,
 
     TXN_NOT_EXISTS = 51,
-    TXN_IN_PROCESSING = 52
+    TXN_IN_PROCESSING = 52,
+
+    SHUTDOWN_TABLET_OCCUPIED = 53
```

Comment:
how about resource busy?

Reply:
good idea, looks more general, i will change it